src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -709,7 +708,7 @@ private void AddTypeKind(INamedTypeSymbol symbol)
         /// <summary>
         /// Copy of <see cref="SynthesizedRecordClone.FindValidCloneMethod(TypeSymbol, ref HashSet{DiagnosticInfo}?)"/>
         /// </summary>
-        private static IMethodSymbol FindValidCloneMethod(ITypeSymbol containingType)
+        internal static IMethodSymbol FindValidCloneMethod(ITypeSymbol containingType)
```

Comment:
>internal [](start = 8, length = 8)

The change could be reverted if TypeSymbol.IsRecord is changes as suggested

Reply:
In fact, now we should be able to remove this method and replace its usage with IsRecord check 

---
In reply to: [494458647](https://github.com/dotnet/roslyn/pull/47960#discussion_r494458647) [](ancestors = 494458647)